packages/react-dom/src/events/__tests__/SyntheticKeyboardEvent-test.js

```diff
@@ -9,114 +9,328 @@
 
 'use strict';
 
-var SyntheticKeyboardEvent;
-var getEventCharCode;
+let React;
+let ReactDOM;
+
+let getEventCharCode;
 
 describe('SyntheticKeyboardEvent', () => {
-  var createEvent;
+  let container;
+
+  beforeEach(() => {    
+    React = require('react');
+    ReactDOM = require('react-dom');
+    // The container has to be attached for events to fire.
+    container = document.createElement('div');
+    document.body.appendChild(container);
 
-  beforeEach(() => {
-    // Mock getEventCharCode for proper unit testing
-    jest.mock('../getEventCharCode');
     getEventCharCode = require('../getEventCharCode').default;
```

Comment:
This is also problematic. It's importing an internal module—which is exactly the situation we're trying to avoid.

Reply:
In fact I think the tests from `getEventCharCode` should just be "rolled" into this test case. Since they're covering the same thing.